frontend/packages/metal3-plugin/src/components/host-menu-actions.tsx

```diff
@@ -0,0 +1,76 @@
+import * as React from 'react';
+
+import { asAccessReview, KebabOption } from '@console/internal/components/utils';
+import { k8sKill, K8sKind, K8sResourceKind } from '@console/internal/module/k8s';
+import { getMachineNodeName, getName } from '@console/shared';
+import { confirmModal } from '@console/internal/components/modals';
+import { NodeMaintenanceModel } from '@console/kubevirt-plugin/src/models';
```

Comment:
Hm, so KubeVirt plugin defines the `NodeMaintenanceModel` but Metal3 plugin realizes the corresponding actions?

Why isn't `NodeMaintenanceModel` moved to Metal3 plugin?

Reply:
ah, you are right; that is much cleaner choice